Module.Build.json

```diff
@@ -4,11 +4,7 @@
       "CoreServicesDlls":[
 		"./Decisions.Financial/obj/Decisions.Financial.dll"
       ],
-      "ReferenceDlls": 
-        [
-          "./Decisions.Financial/obj/ExcelFinancialFunctions.dll",
-          "./Decisions.Financial/obj/FSharp.Core.dll"
```

Comment:
Is FSharp.Core.dll in Decisions/bin already?

Reply:
After I removed ExcelFinancialFunctions from the Module.Build.json file, I got the same conflicting dll error for FSharp.Core. It's not explicitly added in nuget, but brought along by ExcelFinancialFunctions